services/brig/test/integration/API/Team/Util.hs

```diff
@@ -47,22 +57,53 @@ createUserWithTeam brig galley = do
             , "password"        .= defPassword
             , "team"            .= newTeam
             ]
-    bdy <- decodeBody <$> post (brig . path "/i/users" . contentJson . body p)
```

Comment:
No logic changes here. I did these changes because I wanted to return `(User, Team)` instead of `(UserId, TeamId)`, but then it turned out that I had to also change all callsites and that was too much of a hassle.